src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.scss

```diff
@@ -0,0 +1,93 @@
+@import 'src/styles/common';
+
+.table {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr 1fr 2fr;
+  grid-template-rows: auto;
+  min-width: 1900px;
+}
+
+.tableRow {
+  border-bottom: 1px solid $dark-grey;
+}
+
+.tableCell {
+  display: inline-block;
```

Comment:
why is this inline-block and not block?

Reply:
It still works. In fact, both seem to work fine but I agree block would be better.

Reply:
Block will be the default if tableCell is changed from span to div.

Reply:
:heavy_check_mark: